test/util/util.ts

```diff
@@ -1,12 +1,11 @@
-var Vantage = require('../../');
-var _ = require('lodash');
-var path = require('path');
+var Vantage = require("../../dist/vorpal");
```

Comment:
why double quotes?

Reply:
Looks like this got changed on accident. I could go back to PR #2 and implement precise commits with prettier (and rip out Xo) to prevent any mistaken anti-style-guide changes. Thoughts?